test/k8sT/manifests/1.20/demo_ds_dualstack.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: v1
```

Comment:
Consider moving these files from `test/k8sT/manifests/1.20` to `test/k8sT/manifests`. It's likely that these files will work in newer k8s versions and we won't need to add a new directory each time a new k8s version is released.

Reply:
Cool, will do that.